src/algorithms/signal_source/gnuradio_blocks/fifo_reader.h

```diff
@@ -93,6 +67,9 @@ class FifoReader : virtual public gr::sync_block
     const std::string file_name_;
     const std::string sample_type_;
     std::ifstream fifo_;
+    char *fifo_buffer_;
+    int buffer_idx_, buffer_size_;
```

Comment:
please declare each variable in a separate statement. Note that `buffer_size_` is useless information, if you use a container

Reply:
Not applicable anymore after [77c682e](https://github.com/gnss-sdr/gnss-sdr/pull/584/commits/77c682e8b312c645d77e60289b0d67b33e67c633).